pkg/kubectl/cmd/cmd.go

```diff
@@ -397,10 +400,14 @@ func HandlePluginCommand(pluginHandler PluginHandler, cmdArgs []string) error {
 		return nil
 	}
 
+	cmd := exec.Command(foundBinaryPath, cmdArgs[len(remainingArgs):]...)
```

Comment:
why not leave this exec command setup abstracted away in the Execute method of the PluginHandler?

Reply:
I actually had that first :)  and changed it because test started failing. 
exec.Command() copies binary path adds to cmdArgs. If we do the exec setup in Execute,cmdArgs([0]) would not contain binary path. This would mean to change test (TestKubectlCommandHandlesPlugins) to not expect arg[0] as binary path. I preferred to keep test same since it matches with expected behavior.